actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -3,6 +3,7 @@
 # :markup: markdown
 
 require "active_support/core_ext/module/attribute_accessors"
+require "action_controller/metal/rate_limiting"
```

Comment:
Action dispatch should not depend on action controller. Just define the method here. No need to put in `action_controller/metal/rate_limiting`

Reply:
Thank you for that background. I wasn't aware and was following the precedent set in `ActionDispatch::Request`:

https://github.com/rails/rails/blob/9a6054a707c9fe0b0236de08d245a714e4ca73cd/actionpack/lib/action_dispatch/http/request.rb#L9